### PR TITLE
feat: ignore Storybook build folder for Node

### PR DESCRIPTION
**Pull request by caedes**
# Pull Request

Thank you for contributing to @dvcs/gitignore and https://www.gitignore.io.

## New or update

Select the appropriate check box for this pull request. This helps when merging to ensure there are no conflicts with other templates or misunderstandings of how thee template list works.

### New

- [ ] Template - New `.gitignore` template
- [ ] Composition - Template made from smaller templates
- [ ] Inheritance - Template similar to an existing template
- [ ] Patch - Template extending functionality of existing template

### Update

- [x] Template - Update existing `.gitignore` template

## Details

*replace this section with links and/or info about the proposed request*

### CODE REVIEW

**Comment by caedes** [comment]
Seems like it was override by a [previous commit](https://github.com/toptal/gitignore/commit/3c8f56ca701ab051f10f1ea6af66cd708f63b39b#diff-be95ea910b23ee1a6c80490c9e037c4f4b1842ed7c514ec716f483e8e189542d). Maybe because I was treating different tech on the same config file IDK.

File: templates/Node.gitignore [hunk 1]
```diff
@@ -87,6 +87,10 @@ typings/
 .nuxt
 dist
 
+# Storybook build outputs
```
Comment (quibaritaenperdresatrompe) [inline templates/Node.gitignore:90]: It seems that the output directory is `storybook-static/` cf. https://github.com/storybookjs/storybook/blob/c61b404a624d1b18027330906e7cf9000043379e/lib/cli/src/generate.ts#L85

**Review by quibaritaenperdresatrompe — CHANGES_REQUESTED** [review]

File: templates/Node.gitignore (hunk 1 above)
Comment (caedes) [reply templates/Node.gitignore:90]: Done, thx!

**Review by quibaritaenperdresatrompe — APPROVED** [review]

**Comment by gobr** [comment]
@caedes @quibaritaenperdresatrompe thanks!

**Pull request merged**